src/utils/ipfs.ts

```diff
@@ -174,7 +174,7 @@ export async function uploadNftRewardsToIPFS(
 export function formatIpfsLink(ipfsLink: string) {
   const ipfsLinkParts = ipfsLink.split('/')
   const cid = ipfsLinkParts[ipfsLinkParts.length - 1]
-  return `https://${DEFAULT_PINATA_GATEWAY}/ipfs/${cid}`
+  return `https://${IPFS_GATEWAY_HOSTNAME}/ipfs/${cid}`
```

Comment:
No reason for this to use pleb gateway. Might result in the newly uploaded image not showing up. Switched to the private gateway :)

Reply:
If someone inputs their own image URL string, wont this break?

Reply:
Do you mean directly into the memo? If so, it should be fine as this is only touching the upload image input code

Reply:
I see what you're saying @aeolianeth . I've reverted this change